.ci.yaml

```diff
@@ -112,7 +112,7 @@ platform_properties:
         ]
       os: Mac-10.15
       device_os: iOS-14.4.2
-      xcode: 12a7209
+      xcode: 12c33
```

Comment:
I'd recommend to move the xcode change to its own PR to revert easily if something fails.

Reply:
Since this is a no-op PR (until we reland the flutter/infra change), and it's resetting to what is already in starlark, is it ok to keep as a single PR?